test/test-fs-event.c

```diff
@@ -109,6 +109,12 @@ static void fail_cb(uv_fs_event_t* handle,
   ASSERT(0 && "fail_cb called");
 }
 
+static void noop_cb(uv_fs_event_t* handle,
+                    const char* path,
+                    int events,
+                    int status) {
+}
```

Comment:
You're right, we aren't even starting the loop. I guess it's better to just use fail_cb then.